### PR TITLE
Tooling fixes

### PR DESCRIPTION
**Pull request by xaocon**
Thank you for submitting this pull request! We appreciate you spending the time to work on these changes.

## What is the motivation?

It looks like there was a lot mangled during the effort to remove tokio and change the folder layout. I first noticed a merge conflict that got merged in but it send me down a rabbit hole with `poetry` being removed.

## Type of Change

- [ ] 📚 Examples / docs / tutorials / dependencies update
- [x] 🔧 Bug fix (non-breaking change which fixes an issue)
- [ ] 🥂 Improvement (non-breaking change which improves an existing feature)
- [ ] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🔐 Security fix

## What does this change do?

It's primarily clean up of poetry items and correction of the stability action

## What is your testing strategy?

I didn't change any actual code. I tested running ruff, mypy, and black from the cli and pip installed to verify requirements files.

## Is this related to any issues?

There is not

## Have you read the [Contributing Guidelines]?

- [x] I have read the [Contributing Guidelines]

[Contributing Guidelines]: https://github.com/surrealdb/surrealdb/blob/main/CONTRIBUTING.md

### CODE REVIEW

**Comment by xaocon** [comment]
stability should probably also run `cargo fmt` but I didn't want to add a new step in a PR focused on cleanup and fixing previous merge issues. I noticed that the rust code is no longer well formatted. There are also issues in most or all of the current steps in in stability but I think the workflow should function as intended now.

File: .github/workflows/stability.yml [hunk 1]
```diff
@@ -23,28 +23,16 @@ jobs:
...
       - id: ruff
         if: always()
-<<<<<<< HEAD
-        run: poetry run ruff --format=github python_package/surrealdb/
+        run: ruff check --output-format=github surrealdb/
 
       - id: Black
         if: always()
-        run: poetry run black python_package/surrealdb/ --check --verbose --diff --color
+        run: black surrealdb/ --check --verbose --diff --color
```
Comment (gaby) [inline .github/workflows/stability.yml:34]: This should be `ruff format`, no one uses black anymore.

**Comment by tobiemh** [comment]
Closing this pull-request as the latest codebase has now been significantly updated, now making use of a native Python with binary communication with remote databases, and support for embedded database usage. The native Python library now has support for custom types, including datetimes, durations, decimals, geojson types, Record IDs, ranges, and tables 🎉.

**Comment by xaocon** [comment]
Great to hear about all the big changes. I don't want to be pesky but if you aren't using the section of the code base this PR was for then maybe just delete the files and clean up the repo?

**Pull request closed without merging**